site/content/docs/5.1/utilities/text.md

```diff
@@ -101,6 +101,17 @@ Change the line height with `.lh-*` utilities.
 <p class="lh-lg">This is a long paragraph written to show how the line-height of an element is affected by our utilities. Classes are applied to the element itself or sometimes the parent element. These classes can be customized as needed with our utility API.</p>
 {{< /example >}}
 
+## Line length
+
+Limit the line length with `.ll-*` utilities. It applies `max-width` rule to an element with a width based on `ch` dimension, only on block-level elements and not on inline elements.
```

Comment:
`max-width` us a property, not a rule.
`ch` is a unit, not a dimension—and maybe explain why it's done using this unit?
No need to mention `block`/`inline` since `max-width` doesn't apply to `inline` elements anyway. Probably better to mention that it required anything but `inline` to apply.